index.html

```diff
@@ -26,7 +26,6 @@
       <li>Capitol Hill</li>
       <li>Alki</li>
     </ul>
-    <p id="inspiration">Remember! Life might get hard, but always swim upstream!</p>
-    <script src="app.js" type="text/javascript"></script>
+    <p id="inspiration">Remember! Life might get hard, but always swim upstream! Get it? Cause we serve salmon!</p>
```

Comment:
It might be cleaner to format this like

```html
<p id="inspiration">
  Remember! Life might get hard, but always swim upstream! Get it? Cause we serve salmon!
</p>
```